src/main/java/org/broadinstitute/hellbender/utils/pileup/PileupBasedAlleles.java

```diff
@@ -31,47 +54,132 @@
 
             Map<Byte, Integer> altCounts = new HashMap<>();
 
+            int totalAltReads = 0;
+            int totalAltBadReads = 0;
+
             for (PileupElement element : pileup) {
                 final byte eachBase = element.getBase();
 
                 // check to see that the base is not ref and that the alleles are one of these bases - ATCGN
                 // TODO: AH & BG add a better check for acceptable alleles for eachBase
```

Comment:
what's meant by comment? what is insufficient in this check?

Reply:
I think its outdated considering that I have now added checks for "acceptable alleles" with the added heuristics....